package.json

```diff
@@ -191,7 +191,7 @@
     "invariant": "2.2.1",
     "ip": "1.1.3",
     "lodash": "4.15.0",
-    "moment": "^2.14.1",
+    "moment": "^2.15.1",
```

Comment:
It's a good idea to lock dependencies. Remove the `^` in front of the version number, so that everyone get's the same version. Read more about the differences between `^` and `~` [here](http://fredkschott.com/post/2014/02/npm-no-longer-defaults-to-tildes/). 

That article makes the point that 
>Only major versions are allowed to break compatibility, so a developer should be able to switch between minors and patches with ease. The caret trusts that this promise is kept, but the tilde is a little more pessimistic.

Experience tells us that we should be way more pessimistic, and use specific versions, because it reduces the risk of package incompatibility. We can then use tools like Doppins to create pull requests for new versions of the packages. This means there's a build for the pull request, and if it's successful we can just merge it.